chaosazure/__init__.py

```diff
@@ -13,7 +13,7 @@
 from logzero import logger
 from msrestazure.azure_active_directory import AADMixin
 
-from chaosazure.auth import Auth, create
+from chaosazure.authent import Auth, create
```

Comment:
Why did we change it to authent?

Reply:
The chaosazure module offers a function in the init.py - it's called `auth`
The chaosazure module has a module - it's called `auth`

At several cases I was not able to get the desired result - I want to import the chaosazure.auth - the Python compiler does not understand which one to choose.

So this was my attempt to rename it and differntiate the function and the module from each other so that the compiler get what I mean.

Reply:
Ok i get the point. Maybe we have to take a look and check if functions/modules are divided correctly. 
I'm currently occupied with #58 so I will take a look.